src/chrome/content/rules/amd-osx.com.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="AMD OS X">
+	<target host="amd-osx.com" />
+	<target host="*.amd-osx.com" />
```

Comment:
Please avoid wildcard targets if possible. You can simply add `forum.amd-osx.com` and `www.amd-osx.com.

Reply:
Fixed.